frontend/src/components/pages/Account.js

```diff
@@ -121,6 +128,21 @@ export function Account() {
                         </form>
                     </section>
                 );
+            case 'signOutAccount':
+                return (
+                    <section>
+                        <h1 className="text-2xl font-bold mb-4">Signout Of Account</h1>
+                        <p>Click the button below to signout <br />    
```

Comment:
Not sure why you have <br/> here, but removing it shouldn't be a problem.
Also, update signout to log out.